### PR TITLE
implementation of bootstrap table for points

### PR DESCRIPTION
**Pull request by madoleary**
* [x] Are you working on the latest release?
* [x] Have you tested it locally?
* [->] Please explain your change

I got rid of the complex card-based component that was widely used to display points in the application, in favor of a bootstrap table. It makes a big improvement, UX-wise. If this is agreeable, I can implement the table in place of all the other instances of the card-based component in the app (for example, what we use to display services, cases, and topics).

Thanks !

### CODE REVIEW

File: app/views/points/_table.html.erb [hunk 1]
```diff
@@ -1,165 +1,95 @@
...
-</div>
-<!-- card-inline-END -->
-<% end %>
-<% end %>
-</div>
-</div>
-</div>
+  <thead>
+    <tr>
+      <th scope="col" onclick="sortTable(0)">Service</th>
```
Comment (Vinnl) [inline app/views/points/_table.html.erb]: `sortTable` doesn't appear to be implemented?

File: app/views/points/_table.html.erb [hunk 2]
```diff
@@ -1,165 +1,95 @@
...
+      <th scope="col">Title</th>
+      <th scope="col" onclick="sortTable(0)">Rating</th>
```
Comment (Vinnl) [inline app/views/points/_table.html.erb]: `sortTable` doesn't appear to be implemented?

**Comment by madoleary** [comment]
Oh yeah! My bad. They're not implemented. Scraps from when I was playing
with the javascript that sorts by rating. Will remove.

**Comment by Vinnl** [comment]
Odd - I just reviewed the settings, but as far as I can see you should have permissions to merge :S

**Pull request merged**